tensorflow/python/ops/ctc_ops.py

```diff
@@ -656,26 +698,36 @@ def ctc_loss_v2(labels,
       raise ValueError(
           "blank_index must be given when using SparseTensor labels.")
 
+    _ctc_use_cudnn = os.environ.get("TF_CUDNN_CTC_LOSS", "0")
```

Comment:
Environment variables are not ok for configuring TF.

We suggested earlier using the same mechanism we use for cudnn lstm.

Reply:
Hi @alextp , I made an implementation selector version internally but it didn't work. My guess is the mechanism might not work well with the loss function. Contacted with @qlzh727 (the owner of implementation selector) already.